net.sf.eclipsecs.ui/src/net/sf/eclipsecs/ui/CheckstyleUIPluginImages.java

```diff
@@ -101,8 +104,7 @@ public abstract class CheckstyleUIPluginImages {
             "icons/checkstyle_warning.gif"); //$NON-NLS-1$
     MARKER_INFO = AbstractUIPlugin.imageDescriptorFromPlugin(CheckstyleUIPlugin.PLUGIN_ID,
             "icons/checkstyle_info.gif"); //$NON-NLS-1$
-    HELP_ICON = AbstractUIPlugin.imageDescriptorFromPlugin(CheckstyleUIPlugin.PLUGIN_ID,
-            "icons/help.gif"); //$NON-NLS-1$
+    HELP_ICON = PlatformUI.getWorkbench().getSharedImages().getImageDescriptor(ISharedImages.IMG_LCL_LINKTO_HELP);
```

Comment:
Greater than 100 lines. Also see the PR discussion about putting back the configuration.

Reply:
@Bananeweizen, please follow checkstyle

Reply:
@rnveach @romani All the many commits of all my recent PRs _are_ free of checkstyle violations locally. Both in eclipse-cs as well as when running "mvn clean install" on the repository root. I've verified that on 2 different machines. You may want to verify on your local machines maybe. I believe the CI system still gives different results than a local build, even with the patch-filter changes.